src/main/resources/schemas/cop_daily_cause_list.json

```diff
@@ -174,25 +174,6 @@
                 "$id": "#root/locationDetails/region/regionalJOH/items",
                 "type": "object",
                 "properties": {
-                  "johTitle": {
```

Comment:
The COP list does not currently have the isPresiding flag in the schema. Just checking this is expected?

Reply:
Have checked with Chris and the COP list regionalJOH doesn't need the isPresiding flag adding